pylzy/tests/env/test_auto_python_env.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import pytest
+
+from lzy.env.python.auto import AutoPythonEnv
+from lzy.env.explorer.auto import AutoExplorer
+from lzy.env.explorer.base import BaseExplorer, ModulePathsList, PackagesDict
+
+
+def test_explorer_factory():
+    python_env = AutoPythonEnv()
+
+    assert isinstance(python_env._env_explorer, AutoExplorer)
```

Comment:
I'm not sure.
I want to be sure that my default is working and I can't find a better way.

It's okay if it will be broken after refactoring, I think. The repair will not take so much.